mitodl/micromasters
# Remove usages of get_social_username

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #4989 

#### What's this PR do?
This PR removes usages of `get_social_username` in favor of simply using `User.username` so that we can support users with social auth other than edx. I also renamed the existing functions in `profiles/api.py` so it's obvious those are edx-specific.

#### How should this be manually tested?
Generally speaking, nothing should break.

You should still be able to:
- View CMS pages
- Create an account
- Login with an existing account
- View your dashboard
- View your profile
- Edit your profile
- Make a payment
